shell/browser/browser_mac.mm

```diff
@@ -542,4 +542,33 @@ void RemoveFromLoginItems() {
   }
 }
 
+void Browser::RegisterForAPNSNotifications() {
+  [[AtomApplication sharedApplication]
+      registerForRemoteNotificationTypes:NSRemoteNotificationTypeBadge |
+                                         NSRemoteNotificationTypeAlert |
+                                         NSRemoteNotificationTypeSound];
+}
+
+void Browser::UnregisterForAPNSNotifications() {
+  [[AtomApplication sharedApplication] unregisterForRemoteNotifications];
+}
```

Comment:
I don't think there's any reason for these methods to be on `Browser`, you should be able to move them to `PushNotifications` without any issues.